spec/features/admin/accounting/transaction_flow_spec.rb

```diff
@@ -43,11 +43,24 @@
       fill_in 'Date', with: Date.today.to_s
       select accounts.sample.name, from: 'Bank Account'
       fill_in 'Amount', with: '12.34'
-      fill_in 'Description', with: 'Foo bar'
+      fill_in 'Description', with: 'Palm trees'
       fill_in 'Memo', with: 'Chunky monkey'
-      click_on 'Add'
+      page.find('a[data-action="submit"]').click
 
-      expect(page).to have_content("Foo bar")
+      expect(page).to have_content('Palm trees')
     end
   end
+
+  describe 'show', js: true do
+    let!(:loan) { create(:loan) }
+    let!(:txn) { create(:accounting_transaction, project_id: loan.id, description: 'I love icecream') }
+
+    scenario 'can show transactions' do
+      visit admin_loan_tab_path(loan, tab: 'transactions')
+      click_on txn.txn_date.strftime('%B %-d, %Y')
+      sleep 5
```

Comment:
Hooray for feature specs!

This `sleep` should not be necessary though. Capybara should know to wait for the content to appear (does it?). If it isn't, we'd need to troubleshoot that. But using `sleep` in Capybara is an anti-pattern.